src/Datadog.Trace.ClrProfiler.Managed/Helpers/HttpBypassHelper.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+
+namespace Datadog.Trace.ClrProfiler.Helpers
+{
+    internal static class HttpBypassHelper
+    {
+        public static bool ShouldSkipResource(Uri requestUri, string[] patternsToSkip)
```

Comment:
My original assumption in making this is that the defaults would be placed within this method, and this could possibly even live in `ScopeFactory`.
If we are going general purpose like this, then I think this would live better within string extensions.

Reply:
String extensions sounds good. Then it becomes an overload of `String.Contains(string)` which takes multiple search strings instead of one.

Reply:
On second thought, because of the special thought we put into optimizing this, I'd like it to stay separate.
Premature re-use is the root of all evil, or something like that? :P

Reply:
Makes sense, thanks.

Reply:
I still think we can remove the word "skip" from everything here. The method checks if a string matches certain patterns, that's all. It has no idea what that information will be used for. I suggested something like `UrlHelper.Matches(uri, patterns)` originally.